### DISCUSSION TITLE
Change Timezone in Grid and Grafana

### DISCUSSION THREAD

**Discussion by OzzyKampha** [Security-Onion-Solutions/securityonion#8605] [Ideas] [answered]
I would love to change the timezone in the grid and grafana in the same way that the Hunt and Dashboard works today.

**Comment by dougburks** [comment, marked as answer]
We'll consider adding the option to the Grid page:
https://github.com/Security-Onion-Solutions/securityonion/issues/8629

However, for Grafana, you would need to make that request to the Grafana team.
